Readme.md

```diff
@@ -60,3 +61,46 @@ Package |
 [	DotSpatial.Positioning](https://www.nuget.org/packages/DotSpatial.Positioning) |
 [	DotSpatial.Positioning.Forms](https://www.nuget.org/packages/DotSpatial.Positioning.Forms) |
 [	DotSpatial.Positioning.Design](https://www.nuget.org/packages/DotSpatial.Positioning.Design) |
+
+---
```

Comment:
What do you think of moving this to the top of the README or adding a link to this section to the top? Because the README is larger, that might help with the visibility of the "variant" section.

Reply:
Good idea, added link @amaechler.